lighthouse-core/fraggle-rock/config/default-config.js

```diff
@@ -98,7 +98,6 @@ const defaultConfig = {
     {id: artifacts.EmbeddedContent, gatherer: 'seo/embedded-content'},
     {id: artifacts.FontSize, gatherer: 'seo/font-size'},
     {id: artifacts.Inputs, gatherer: 'inputs'},
-    {id: artifacts.FullPageScreenshot, gatherer: 'full-page-screenshot'},
```

Comment:
This is actually a minor bug fix for timespan/snapshot because they weren't doing FPS as the final gatherer before.